src/lib.rs

```diff
@@ -247,10 +255,6 @@ impl Builder {
             output_vector.push("--no-derive-debug".into());
         }
 
-        if !self.options.impl_debug {
```

Comment:
Is suppose this was a bug...

Reply:
This should be `if self.options.impl_debug { ... }`; we still want to add it to the output when set. Care to do this in a new PR?

Reply:
Err, I see you did that in another place. Want to submit that as a new PR or at least make it its own standalone commit?

Reply:
Yeah, I thought I would rebase & squash, but now I see that this PR becoming too big.